### ISSUE TITLE
Migrating to backstage community-plugins

### ISSUE THREAD

**Issue by RedlineTriad** [arhill05/backstage-plugin-scaffolder-git-actions#7]
Hey @arhill05 would you be interested in migrating your plugin to the community repository?
https://github.com/backstage/community-plugins?tab=readme-ov-file#contributing-a-plugin

This would make contribution and maitenance easier.

I would also be willing to do this myself, though I would like your approval first so we avoid forks.

**Comment by arhill05** [comment]
Hey there! I'm totally okay with moving it, and I appreciate you offering to take it on.

I'm currently on leave from work so I wouldn't have much time to do it now anyway 🙂

**Comment by isaque-r-silva** [comment]
Hi @RedlineTriad, started the migration?

**Comment by RedlineTriad** [comment]
> Hi @RedlineTriad, started the migration?

Nope, am on vacation and this would be mostly useful for my employer.
Unlikely to get to this in the next few weeks after getting back to work.
